src/Starcounter.Database.Extensions/OnCommitTransactor.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Starcounter.Database.ChangeTracking;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Starcounter.Database.Extensions
+{
+    public abstract class OnCommitTransactor<TTransactorContext> : TransactorBase<TTransactorContext>
```

Comment:
Is inheritance justified here? No polymorphic usage. No shared state either. I suppose a static helper would suffice.

Reply:
Static helper seems to be an ugly solution when we try to have everything done through DI.

Reply:
Well, it's unrelated as I see it. We don't inject base classes. A static helper is just a code that you want to reuse. There is nothing wrong with it.

Reply:
Updated.